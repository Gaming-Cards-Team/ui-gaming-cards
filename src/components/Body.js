import React from "react";
import Banner from "./Banner";
import TopMessage from "./TopMessage";
import CardGallery from "./CardGallery";

const Body = () => {
  return (
    <div>
      <Banner></Banner>
      <TopMessage></TopMessage>
     
      <CardGallery></CardGallery>
    </div>
  );
};

export default Body;
