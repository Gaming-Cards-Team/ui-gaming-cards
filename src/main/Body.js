import React from 'react';

const Body = () => (
    <div id='body'>
       Body
    </div>
);

export default Body;