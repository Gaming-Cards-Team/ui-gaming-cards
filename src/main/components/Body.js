import React from "react";
import Banner from "./Banner";

const Body = () => {
  return (
      <Banner></Banner>
  );
};

export default Body;
