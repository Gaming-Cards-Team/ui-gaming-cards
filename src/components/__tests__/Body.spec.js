import React from 'react';
import Body from '../Body';

import renderer from 'react-test-renderer';

describe('Test Body component', () => {
    it('should validate the component', () => {
        const body = renderer.create(<Body />).toJSON();

        expect(body).toMatchSnapshot();
    });
});