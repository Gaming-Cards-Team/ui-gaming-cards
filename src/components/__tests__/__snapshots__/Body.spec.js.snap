// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Body component should validate the Body component 1`] = `
<div>
  <img
    alt="Star Wars Banner"
    className=""
    data-testid="banner"
    src="banner.png"
    width="100%"
  />
  <div
    style={
      Object {
        "background": "black",
        "color": "white",
        "fontSize": "larger",
        "padding": "10px 30px",
        "width": "100%",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col-9"
        style={
          Object {
            "padding": "10px 0px",
          }
        }
      >
        Are you looking for cards for other game cards? check our catalog
        <a
          href="https://www.youtube.com/"
          style={
            Object {
              "color": "#2d963f",
              "fontWeight": "bold",
            }
          }
        >
           here
        </a>
      </div>
      <div
        className="col-3"
        style={
          Object {
            "textAlign": "right",
          }
        }
      >
        <a
          href="https://www.youtube.com/watch?v=3gPu9pp2WOM"
          style={
            Object {
              "color": "#2d963f",
              "fontWeight": "bold",
            }
          }
        >
          How to play
          <img
            alt="Level Up Logo"
            data-testid="youtubeVideo"
            height="48"
            src="youtubeVideoImage.png"
          />
        </a>
      </div>
    </div>
  </div>
  <div
    style={
      Object {
        "marginBottom": 50,
        "marginLeft": 100,
        "marginRight": 100,
        "marginTop": 30,
      }
    }
  >
    <div
      className="card-deck"
    >
      <div
        className="card"
      >
        <img
          className="card-img-top"
          src="https://swdestinydb.com/bundles/cards/en/01/01001.jpg"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            Captain Phasma
          </div>
          <p
            className="card-text"
          >
            <div
              style={
                Object {
                  "textAlign": "right",
                  "widht": "100%",
                }
              }
            >
              $1.00
            </div>
          </p>
        </div>
        <div
          className="card-footer"
        >
          <small
            className="text-muted"
          >
            Add to cart
          </small>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          className="card-img-top"
          src="https://swdestinydb.com/bundles/cards/en/01/01002.jpg"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            First Order Stormtrooper
          </div>
          <p
            className="card-text"
          >
            <div
              style={
                Object {
                  "textAlign": "right",
                  "widht": "100%",
                }
              }
            >
              $1.00
            </div>
          </p>
        </div>
        <div
          className="card-footer"
        >
          <small
            className="text-muted"
          >
            Add to cart
          </small>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          className="card-img-top"
          src="https://swdestinydb.com/bundles/cards/en/01/01003.jpg"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            General Grievous
          </div>
          <p
            className="card-text"
          >
            <div
              style={
                Object {
                  "textAlign": "right",
                  "widht": "100%",
                }
              }
            >
              $1.00
            </div>
          </p>
        </div>
        <div
          className="card-footer"
        >
          <small
            className="text-muted"
          >
            Add to cart
          </small>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          className="card-img-top"
          src="https://swdestinydb.com/bundles/cards/en/01/01004.jpg"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            General Veers
          </div>
          <p
            className="card-text"
          >
            <div
              style={
                Object {
                  "textAlign": "right",
                  "widht": "100%",
                }
              }
            >
              $1.00
            </div>
          </p>
        </div>
        <div
          className="card-footer"
        >
          <small
            className="text-muted"
          >
            Add to cart
          </small>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          className="card-img-top"
          src="https://swdestinydb.com/bundles/cards/en/01/01005.jpg"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            AT-ST
          </div>
          <p
            className="card-text"
          >
            <div
              style={
                Object {
                  "textAlign": "right",
                  "widht": "100%",
                }
              }
            >
              $1.00
            </div>
          </p>
        </div>
        <div
          className="card-footer"
        >
          <small
            className="text-muted"
          >
            Add to cart
          </small>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          className="card-img-top"
          src="https://swdestinydb.com/bundles/cards/en/01/01006.jpg"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            First Order TIE Fighter
          </div>
          <p
            className="card-text"
          >
            <div
              style={
                Object {
                  "textAlign": "right",
                  "widht": "100%",
                }
              }
            >
              $1.00
            </div>
          </p>
        </div>
        <div
          className="card-footer"
        >
          <small
            className="text-muted"
          >
            Add to cart
          </small>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          className="card-img-top"
          src="https://swdestinydb.com/bundles/cards/en/01/01007.jpg"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            Commanding Presence
          </div>
          <p
            className="card-text"
          >
            <div
              style={
                Object {
                  "textAlign": "right",
                  "widht": "100%",
                }
              }
            >
              $1.00
            </div>
          </p>
        </div>
        <div
          className="card-footer"
        >
          <small
            className="text-muted"
          >
            Add to cart
          </small>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          className="card-img-top"
          src="https://swdestinydb.com/bundles/cards/en/01/01008.jpg"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            F-11D Rifle
          </div>
          <p
            className="card-text"
          >
            <div
              style={
                Object {
                  "textAlign": "right",
                  "widht": "100%",
                }
              }
            >
              $1.00
            </div>
          </p>
        </div>
        <div
          className="card-footer"
        >
          <small
            className="text-muted"
          >
            Add to cart
          </small>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          className="card-img-top"
          src="https://swdestinydb.com/bundles/cards/en/01/01009.jpg"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            Count Dooku
          </div>
          <p
            className="card-text"
          >
            <div
              style={
                Object {
                  "textAlign": "right",
                  "widht": "100%",
                }
              }
            >
              $1.00
            </div>
          </p>
        </div>
        <div
          className="card-footer"
        >
          <small
            className="text-muted"
          >
            Add to cart
          </small>
        </div>
      </div>
      <div
        className="card"
      >
        <img
          className="card-img-top"
          src="https://swdestinydb.com/bundles/cards/en/01/01010.jpg"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            Darth Vader
          </div>
          <p
            className="card-text"
          >
            <div
              style={
                Object {
                  "textAlign": "right",
                  "widht": "100%",
                }
              }
            >
              $1.00
            </div>
          </p>
        </div>
        <div
          className="card-footer"
        >
          <small
            className="text-muted"
          >
            Add to cart
          </small>
        </div>
      </div>
    </div>
  </div>
</div>
`;
